protos/flyteidl/admin/quality_of_service.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+package flyteidl.admin;
+option go_package = "github.com/lyft/flyteidl/gen/pb-go/flyteidl/admin";
+
+
+message QualityOfServiceSpec {
+  // Indicates how much queueing delay an execution can tolerate.
+  uint32 queueing_budget_mins = 1;
```

Comment:
Should we use [this](https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Duration) instead?

Reply:
originally that's what I had but people commented that anything beyond minutes was too much granularity and unnecessary

one option is we could expose it as queueing_budget_mins in the SDK and then store that as a duration in the proto field but that can still be manipulated by non-SDK users

Reply:
will go with the second option, updated to use duration and we can expose it as a field in mins only in the sdk cc @migueltol22